networkx/tests/test_convert_pandas.py

```diff
@@ -34,40 +34,69 @@ def __init__(self, ):
     def assert_equal(self, G1, G2):
```

Comment:
```is_isomorphic``` is much harder to test than what our ```assert_graphs_equal``` tests.
It looks like the current code doesn't actually need the is_isomorphic though.  
Those graphs really are the same -- not just isomorphic.  If that's true, then yes -- please use ```assert_graphs_equal```.